src/views/settings/Auth/Basic.vue

```diff
@@ -55,9 +55,9 @@ export default {
       authMethodsSetting: state => state.settings.authMethods
     })
   },
-  mounted() {
-    this.$store.dispatch('settings/getAuthMethods')
-    this.initAuthItems()
+  async mounted() {
+    await this.$store.dispatch('settings/getAuthMethods')
+    await this.initAuthItems()
   },
   methods: {
     async initAuthItems() {
```

Comment:
There is no code provided to examine for irregularities, potential issues or optimization suggestions. Please provide the code so that I can assist you properly.